doc/transform_methods/hansenlaw.rst

```diff
@@ -121,6 +121,19 @@ If you would like to access the Hansen-Law algorithm directly (to transform a
 right-side half-image), you can use :func:`abel.hansenlaw.hansenlaw_transform`.
 
 
+
+Tips
+----
+
+`hansenlaw` tends to perform better with images of large size :math:`n \gt 1001` pixel width. For smaller images the angular_integration (speed) profile may look better if sub-pixel sampling is used via: ::
+
+    transform_options=dict(dr=0.1)
```

Comment:
Tricky, and somewhat confusing.  

Part of this issue arises because my own research processes images that may be `x0.5` ... `x4` in size, in which case `dr` gives the same final intensity scaling, independent of the image size.  ie. my data (measured using a `2048x2048` pixel camera) is binned to images of size `1024x1024` ... `8192x8192`, the former maximize intensity, the later size resolution.

I am probably unique in this. In general, you are right `dr=1` or the actual physical scaling parameter. 

I will change the examples, docs, tonight.